frontend/copr-frontend.spec

```diff
@@ -4,7 +4,7 @@
 # https://fedoraproject.org/wiki/Packaging:Guidelines#Packaging_of_Additional_RPM_Macros
 %global macrosdir       %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 
-%global copr_common_version 0.19.1~~dev
+%global copr_common_version 0.24.1.dev0
```

Comment:
Shouldn't we keep the `~~` requirement here?

Reply:
I found it strange that backend and frontend used a different version format. Backend:

```
%global copr_common_version 0.21.1.dev1
```

and frontend:

```
%global copr_common_version 0.19.1~~dev
```

Since backend had a newer version, I thought that's how we do it now.